app/plugin.go

```diff
@@ -363,11 +526,15 @@ func (a *App) DisablePlugin(id string) *model.AppError {
 }
 
 func (a *App) InitPlugins(pluginPath, webappPath string, supervisorOverride pluginenv.SupervisorProviderFunc) {
-	if !*a.Config().PluginSettings.Enable {
+	if a.PluginEnv != nil {
```

Comment:
Changed up the order here to avoid doing anything if the plugin environment cannot be initialized. This is another area for which I need to add reporting, though in this case it's really a top-level application error vs. a plugin status.